.github/workflows/format_json.yaml

```diff
@@ -1,7 +1,9 @@
 name: 'PBIX Deserializer'
 description: 'Deserializes PBIX Files into json files for easier version control'
-runs:
-  using: "composite"
+on: pull_request
+jobs:
+    Reformat-PBIX-Files:
+runs-on: ubuntu-latest
```

Comment:
The tabbing is off, that's why it's crashing. Yaml is very white space sensitive.

You can paste your yaml here to verify that it is formatted correctly. http://www.yamllint.com/

Reply:
Python is also very white space sensitive so you'll need to get used to it.